src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -126,15 +127,20 @@ internal static Encoding GetDefaultEncoding()
         /// </summary>
         internal static Encoding GetOEMEncoding()
         {
+            // The OEM code pages are sometimes used by Win32 console applications, and
+            // on non-Windows platforms they still may have uses (if installed) and
+            // could be used if desired.
+            // On non-windows platforms, they have more limited uses, and probably won't
```

Comment:
This sentence is a bit confusing. Doesn't the previous one say everything that needs to be said?